package.json

```diff
@@ -1,71 +1,70 @@
+    "install-dependencies": "if [ ! -d node_modules ]; then npm install && npx npm-install-peers; fi",
+    "prebuild": "npm run install-dependencies",
+    "build": "rollup -c scripts/rollup.config.js",
+    "prestorybook": "npm run install-dependencies",
+    "storybook": "start-storybook -p 6006",
+    "storybook:build": "build-storybook",
+    "storybook:deploy": "gh-pages -d storybook-static -m 'Update GitHub Pages [ci skip]'",
+    "test": "CI=true react-scripts test --env=jsdom"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/italia/design-react.git"
-  },
-  "author": {
-    "name": "RomaJS",
-    "url": "http://romajs.org/"
+    "url": "git+https://github.com/italia/design-react-kit.git"
   },
-  "contributors": [
-    "Matteo Manchi <matteo.manchi@gmail.com>",
-    "Marco Liberati <dej611@gmail.com>",
-    "Paolo Mariotti <mariotti.pro@gmail.com>"
-  ],
-  "license": "LGPLv2.1",
   "bugs": {
-    "url": "https://github.com/italia/design-react/issues"
+    "url": "https://github.com/italia/design-react-kit/issues"
+  },
+  "homepage": "https://github.com/italia/design-react-kit#readme",
+  "peerDependencies": {
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0"
   },
-  "homepage": "https://github.com/italia/design-react#README",
   "devDependencies": {
-    "@improntaadvance/eslint-config": "^0.8.2",
-    "@storybook/addon-actions": "^3.2.6",
-    "@storybook/addon-console": "^1.0.0",
-    "@storybook/addon-info": "^3.2.9",
-    "@storybook/addon-storyshots": "^3.2.12",
-    "@storybook/react": "^3.2.8",
-    "@storybook/storybook-deployer": "^2.0.0",
+    "@sambego/storybook-state": "^1.0.7",
+    "@storybook/addon-knobs": "^3.4.2",
+    "@storybook/addon-options": "^3.4.2",
+    "@storybook/addon-storyshots": "^3.4.2",
+    "@storybook/addon-viewport": "4.0.0-alpha.1",
+    "@storybook/addons": "^3.4.2",
+    "@storybook/react": "^3.4.2",
+    "babel-core": "^6.26.0",
     "babel-eslint": "^7.2.3",
+    "babel-plugin-external-helpers": "^6.22.0",
     "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-preset-airbnb": "^2.4.0",
-    "eslint": "^4.8.0",
-    "eslint-config-prettier": "^2.6.0",
-    "eslint-plugin-flowtype": "^2.39.1",
-    "eslint-plugin-import": "^2.7.0",
-    "eslint-plugin-jsx-a11y": "^5.1.1",
-    "eslint-plugin-prettier": "^2.3.1",
-    "eslint-plugin-react": "^7.4.0",
-    "extract-text-webpack-plugin": "^3.0.1",
-    "jest": "^21.2.1",
-    "prettier": "^1.7.4",
-    "raw-loader": "^0.5.1",
-    "react-test-renderer": "^15.6.1"
-  },
-  "dependencies": {
-    "classnames": "^2.2.5",
-    "design-web-toolkit": "github:italia/design-web-toolkit",
-    "jquery": "^3.2.1",
-    "react": "^15.6.1",
-    "react-dom": "^15.6.1",
-    "tablesaw": "^3.0.3"
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-preset-env": "^1.6.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-runtime": "^6.26.0",
+    "bootstrap-italia": "^0.7.1",
```

Comment:
Questo non dovrebbe far parte delle `dependencies` o almeno delle `peerDependencies`?

Reply:
In teoria è usato solo per le stories, un'app di terze parti lo importerebbe puntualmente (tramite CSS o SCSS).

Reply:
Ma quindi il tema css non dipende da lui?
Perchè a me suona come `peerDepencency`.

Reply:
o devDependency

Reply:
E' attualmente una `devDependencies` poichè viene usato per le stories di `Storybook`. 
E' di fatto una `peerDepencency` ma per quel che riguarda lo styling. Teoricamente il consumer potrebbe decidere di costruirsi il suo tema, buildarlo in locale (sovrascrivendo le variabili) o importarne uno realizzato da terzi, lasciando quindi inutilizzata la `depencency`.

Reply:
Come mi faceva notare @mjhost forse vale la pena considerare `react` e affini in `peer`.
Magari vediamo questo prima a questo punto.

## Discussione minore

Per lo UI-Kit capisco il tuo punto: personalmente mi piacerebbe avere uno strumento che mi avvisi se non ho lo UI Kit installato quando provo ad utilizzare `design-react-italia`. Con `peerDependencies` ho questo servizio di controllo.
Per quel che riguarda la personalizzazione, c'è sempre il sistema via SASS che permette di modificare le variabili IMHO per il tema.

Se poi l'utente volesse utilizzare gli stessi componenti dello UI-Kit senza utilizzare affatto il suo stile, ma scrivendosi uno stile completamente da capo a mano, lo trovo, credo, poco probabile.

Reply:
In https://github.com/italia/design-react-kit/blob/master/package.json (che viene qui mergiato) c'è
```
  "peerDependencies": {
    "react": "^16.0.0",
    "react-dom": "^16.0.0"
  },
```

Il lavoro attuale è di utilizzare [Bootstrap Italia](https://github.com/italia/bootstrap-italia) al posto dello UI Kit. Teoricamente il tema è ok "istituzionalmente" ma richiede customizzazioni dai consumers (i.e. comuni, scuole, etc) cc @francescozaia.

Reply:
Per contestualizzare (chiarire) la scelta degli stili, faccio un paragone con librerie "diffuse":

https://github.com/JedWatson/react-select#installation
```js
// You can then import react-select and its styles in your application as follows:

import Select from 'react-select';
import 'react-select/dist/react-select.css';
```

https://reactstrap.github.io/
```js
// Install reactstrap and Bootstrap from NPM. Reactstrap does not include Bootstrap CSS so this needs to be installed as well:

// npm install bootstrap --save
// npm install --save reactstrap react react-dom
// Import Bootstrap CSS in the src/index.js file:

import 'bootstrap/dist/css/bootstrap.min.css';
```

Reply:
anche reactstrap va spostato, sicuramente non suona come una devdependency